node/client/Cargo.toml

```diff
@@ -2,19 +2,16 @@
 name = "cord-client"
 description = "Cord Client."
 version = "0.8.1"
-authors = ['Dhiway <info@dhiway.com>',]
+authors = ['Dhiway <info@dhiway.com>']
 edition = '2021'
 license = "GPL-3.0"
 homepage = "https://cord.network"
 repository = "https://github.com/dhiway/cord"
 
 [dependencies]
-async-trait = "0.1.57"
 futures = "0.3.21"
 frame-benchmarking = { git = "https://github.com/dhiway/substrate", branch = "release-v0.9.43" }
 frame-benchmarking-cli = { git = "https://github.com/dhiway/substrate", branch = "release-v0.9.43" }
-pallet-transaction-payment = { git = "https://github.com/dhiway/substrate", branch = "release-v0.9.43" }
```

Comment:
Question: Is transaction payment (Token) removed from the chain? For every signed transaction, an exchange of tokens (WAY) happens. Is it now removed?

Reply:
Payment is removed. Balance is still required to enable the Genesis accounts. We can remove this requirement in a future PR.

Reply:
yes, it is removed.

Reply:
Yes, the 'transaction' payment is not present anymore, reducing number of events, and thus increasing the performance of the pallet's methods.

It will be truely based on authority now.